server/Makefile

```diff
@@ -0,0 +1,19 @@
+REPORTER ?= spec
+NODE_BIN := $(shell npm bin)
+BASE_DIR := $(NODE_BIN)/../..
+NODE_PATH := $(BASE_DIR)/test:$(BASE_DIR)/server:$(BASE_DIR)/client
+BABEL := $(NODE_BIN)/babel
+
+test: i18n/test/out
+	@NODE_ENV=development NODE_PATH=$(NODE_PATH) $(BASE_DIR)/test/runner.js --reporter=$(REPORTER) $(addprefix --grep ,$(GREP))
+
+test-%: i18n/test/out
+	@GREP="$*" make test
+
+i18n/test/out: i18n/test/examples
```

Comment:
Either way, added it back in https://github.com/Automattic/wp-calypso/commit/a7d8c2c0157fe72cade3fe70bb21784f1a480052